CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 ### ...
 
+## 1.1.0 (02.07.2019)
+
+### Enhancements
+
+#### ([1033](https://github.com/allegro/hermes/pull/1033)) Zookeeper multi datacenter
+
+Starting from this release it's possible to run Hermes on multiple independent Zookeeper clusters.
```

Comment:
I would add something like this: Changes are only in hermes-management module which from now on can operate on multiple Zookeeper clusters.